dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -227,24 +227,35 @@ Sound.prototype.newPlayableBufferSource = function(buffer, options) {
   var newSound = this.audioContext.createBufferSource();
```

Comment:
It looks like our audio system has lived in dashboard and been used by apps for a while now. @bcjordan knows more about this than I do.